.travis.yml

```diff
@@ -33,12 +33,16 @@ install:
   # Install all Python dependencies
   - conda install -q --yes python=$TRAVIS_PYTHON_VERSION mkl --file req-travis-conda.txt
   - pip install -q -r req-travis-pip.txt
+  - pip install -q git+git://github.com/mila-udem/blocks-extras.git 
```

Comment:
There seeems to be a syntax to install extra dependencies of a package, e.g. see the command `git+git@github.com:USER/blocks.git#egg=blocks[test,docs] ` from [Blocks installation tutorial](http://blocks.readthedocs.org/en/latest/setup.html). It could be used to make sure that `bokeh` is installed automatically with `blocks-extras`.